.github/workflows/testing.yml

```diff
@@ -38,6 +38,24 @@ jobs:
           command: lint
           config: .github/ct.yaml
 
+  unit-test:
```

Comment:
any reason of adding this job?
https://github.com/hellofresh/kangal/pull/24/files#diff-9d29531f011d792c2f2f2c87ac68865fR60
already provides you coverage

Reply:
Example: https://github.com/hellofresh/kangal/runs/1166481484?check_suite_focus=true#step:7:15